### PR TITLE
[8.x.x] Develop ontimize-web-ngx-map documentation

### PR DESCRIPTION
**Pull request by pacogomezimatia**

### CODE REVIEW

File: _ontimize_web_ngx_map/configuration/04-o-geojson-layer.component-api.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+permalink: /map/configuration/o-geojson-layer/api
```
Comment (patricia-martinez-imatia) [inline _ontimize_web_ngx_map/configuration/04-o-geojson-layer.component-api.md]: I think this file isnt necessary and this page shout use other layout (without API tab)

File: _ontimize_web_ngx_map/configuration/01-o-map-draw-controls.component.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+title: "o-map-draw-controls"
+comp: omapdrawcontrols
+layout: o-component
+
+---
+
+## Introduction
+
+
+The component *o-map-draw-controls* is used to show a draw toolbar into the map. You can catch the events triggered with
```
Comment (manuel-cid) [inline _ontimize_web_ngx_map/configuration/01-o-map-draw-controls.component.md]: Is this complete?

File: _ontimize_web_ngx_map/configuration/04-o-geojson-layer.component.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+
+## Introduction
+
+
+You can add multiple WMS layers to your map.
+
+
+## Basic Example
+
+This is a basic example of a *Map* using the component *o-map-layer* inside to add new WMS layers to the map.
```
Comment (manuel-cid) [inline _ontimize_web_ngx_map/configuration/04-o-geojson-layer.component.md]: This is the GeoJson layer documentation file but the description is about WMS layers.

**Review by manuel-cid — CHANGES_REQUESTED** [review]

File: _data/components/omaplayer.yml [hunk 4]
```diff
@@ -0,0 +1,97 @@
+directive : "o-map-layer"
+
+attributes: [{
+  name: "layer-center",
+  description: "Center the position of the layer",
```
Comment (daniel-grana) [inline _data/components/omaplayer.yml]: Explain that the structure is latitude,longitude separated by comma and maybe give an example

File: _data/components/omarker.yml [hunk 5]
```diff
@@ -0,0 +1,97 @@
+directive : "o-map-layer"
+
+attributes: [{
+  name: "layer-center",
+  description: "Center the position of the layer",
```
Comment (daniel-grana) [inline _data/components/omarker.yml]: Explain that the structure is latitude,longitude separated by comma

File: _data/components/omarker.yml [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+  type: "string"
+},{
+  name: "layer-points",
+  description: "Initial map zoom level.",
```
Comment (daniel-grana) [inline _data/components/omarker.yml]: This is not the correct definition

File: _data/components/omarker.yml [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+  type: "string"
+},{
+  name: "layer-radius",
+  description: "Minimum zoom level of the map.",
```
Comment (daniel-grana) [inline _data/components/omarker.yml]: This is not the correct definition

File: _data/components/omarker.yml [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+  type: "string"
+},{
+  name: "layer-bounds",
+  description: "Maximum zoom level of the map.",
```
Comment (daniel-grana) [inline _data/components/omarker.yml]: This is not the correct definition

File: _data/components/omarker.yml [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+  type: "string"
+},{
+  name: "layer-service",
+  description: "Show or not zoom controls.",
```
Comment (daniel-grana) [inline _data/components/omarker.yml]: This is not the correct definition

File: _data/components/omarker.yml [hunk 10]
```diff
@@ -0,0 +1,97 @@
...
+  type: "boolean",
+  default: true
+},{
+  name: "layer-base-url",
+  description: "Show or not search control.",
```
Comment (daniel-grana) [inline _data/components/omarker.yml]: This is not the correct definition

File: _data/components/omarker.yml (hunk 8 above)
Comment (daniel-grana) [reply _data/components/omarker.yml]: It would be useful an example

File: _data/components/overviewmap.yml [hunk 11]
```diff
@@ -0,0 +1,142 @@
+directive : "o-map"
+
+attributes: [{
+  name: "center",
+  description: "Initialize the map with a given center in latitude and longitude",
```
Comment (daniel-grana) [inline _data/components/overviewmap.yml]: Add an example to see how to configure it

File: _data/components/overviewmap.yml [hunk 12]
```diff
@@ -0,0 +1,142 @@
...
+  name: "draw-control",
+  description: "Enable or disable draw controls",
+  type: "string"
+},{
+  name: "layer-panel-visible",
+  description: "Type of data displayed on Y axis",
+  type: "string"
+},{
+  name: "base-layer-ids",
+  description: "Defines the array of base layer ids",
```
Comment (daniel-grana) [inline _data/components/overviewmap.yml]: Maybe an example would be useful

File: _data/components/owmslayer.yml [hunk 13]
```diff
@@ -0,0 +1,97 @@
+directive : "o-map-layer"
+
```
Comment (daniel-grana) [inline _data/components/owmslayer.yml]: Please check description definitions

**Review by daniel-grana — CHANGES_REQUESTED** [review]

**Pull request merged**